modules/webbrowser/interaction/cefinteractionhandler.cpp

```diff
@@ -52,6 +54,18 @@ void CEFInteractionHandler::invokeEvent(Event* event) {
             event->markAsUsed();
             break;
         }
+        case CanvasHideEvent::chash(): {
```

Comment:
I don't think this is a good way of doing this, what if more then one canvas is attached? what about other types of sinks?
Should really check if there is a connected sink... only the networkevaluator really knows this right now. But I you want we might be able to expose that...

Reply:
Good point! Better to use onProcessorReadyChanged?

Reply:
What you could do now is to add your self as an observer to the processor network on processor added removed.
then you would iterate over all you out connected processors add your self as observer to them and override the onProcessorSinkChanged, then you can respond whenever the sink status of your dependent processors change, which determines whether you should be evaluating or not...

Reply:
If you want I can explain further in person tomorrow. We chould also discuss if the "will be evaluated ness" of a processor should/needs to be exposed in a easier way...